build.proj

```diff
@@ -62,7 +63,18 @@
     <MSBuild Projects="@(PkgProject)"
              Targets="Pack" />
   </Target>
-
+  
+  <ItemGroup>
+     <TestFile Include="d" Url="https://archive.ics.uci.edu/ml/machine-learning-databases/wine-quality/winequality-white.csv"/>
+  </ItemGroup>
+  
+  <Target Name="DownloadExternalTestFiles" Condition="'$(RunTests)'=='true'">
+    <Message Importance="High" Text="Downloading external test files..." />
+      <DownloadFilesFromUrl Items="@(TestFile)"
```

Comment:
Is there a reasonable way to check the file hash? At some point the UCI site will be broken, and/or we will receive (perhaps) a partial file.

As we expand on the use of this, there'll be additional ways for it to fail.